packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -59,6 +65,37 @@ def test_if_marathon_app_can_be_deployed_with_mesos_containerizer(cluster):
     cluster.marathon.deploy_test_app_and_check(app, test_uuid)
 
 
+def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(cluster):
```

Comment:
Hey @mellenburg, this test is a bit tricky because when it gets run as part of the Enterprise integration tests, we expect it to fail only in strict security mode. I'm trying to figure out how we can specify that expected failure here, or perhaps there is a way to override this test in Enterprise? Any thoughts?